cmd/namespace/namespace.go

```diff
@@ -97,6 +97,26 @@ func RunNamespace(ctx context.Context, namespace string) error {
 	return nil
 }
 
+//CleanNamespace cleans a namespace locally
+func CleanNamespace(ctx context.Context, namespace string) error {
+	cred, err := okteto.GetCredentials(ctx, namespace)
+	if err != nil {
+		return err
+	}
+
+	kubeConfigFile := config.GetKubeConfigFile()
+
+	u, _ := url.Parse(okteto.GetURL())
+	parsedHost := strings.ReplaceAll(u.Host, ".", "_")
+
+	if err := okteto.RemoveKubeConfig(cred, kubeConfigFile, namespace, okteto.GetUserID(), parsedHost); err != nil {
+		return err
+	}
+
+	log.Success("Cleaned context '%s' in '%s'", parsedHost, kubeConfigFile)
```

Comment:
```suggestion
	log.Success("Removed context '%s' from '%s'", parsedHost, kubeConfigFile)
```

Reply:
Should I rename everything to remove instead of clean? That seems to make more sense

Reply:
Yeah, that would be great.